pkg/combustion/combustion.go

```diff
@@ -163,6 +163,17 @@ func (c *Combustion) Configure(ctx *image.Context) error {
 		combustionScripts = append(combustionScripts, scripts...)
 	}
 
+	// We manually add the cleanup component as to always make sure it is last
+	cleanupComponent := componentWrapper{
```

Comment:
It feels a bit overkill to mimic the wrapper struct we use for the full list of them. Am I missing something, or can you simplify it to:

```
s, err := configureCleanup(ctx)
if err != nil {
    return fmt.Errorf("configuring cleanup component %q: %w", cleanupComponentName, err)
}
combustionScripts = append(combustionScripts, s...)
```

Reply:
That works